src/views/guidelines/guidelines.jsx

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+var FormattedMessage = require('react-intl').FormattedMessage;
+
+var render = require('../../lib/render.jsx');
+var Page = require('../../components/page/page.jsx');
+var Box = require('../../components/box/box.jsx');
+
+require('./guidelines.scss');
+
+var Guidelines = React.createClass({
+    type: 'Guidelines',
+    render: function () {
+        return (
+            <Box title={<FormattedMessage id='guidelines.title' />}>
+                <div className="bottom">
+                    <p><FormattedMessage id='guidelines.header' /></p>
```

Comment:
I think it'd be good to add `className="intro"` to this `p` tag. That'll make the text slightly more pronounced than the text below.